experimental/dds/tree2/src/feature-libraries/typed-schema/index.ts

```diff
@@ -32,7 +33,7 @@ export {
 	validateStructFieldName,
 	SchemaLibraryData,
 	SchemaLintConfiguration,
-	buildViewSchemaCollection,
+	aggregateSchemaLibraries as buildViewSchemaCollection,
```

Comment:
Is this aliasing desired? Or just VSCode trying to be helpful while refactoring?

Reply:
Fixed!